src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_format.py

```diff
@@ -189,9 +190,11 @@ def _get_difference(finishTime, startTime):
     date_finish = datetime.strptime(temp, "%y-%m-%d %H:%M:%S")
     date_start = datetime.strptime(startTime.replace("T", " ")[2:19], "%y-%m-%d %H:%M:%S")
     ret = date_finish - date_start
-    return "{0}s".format(ret.seconds)
+    hours = "{0:02d}".format((24 * ret.days) + (ret.seconds // 3600))
+    minutes = "{0:02d}".format((ret.seconds % 3600) // 60)
+    seconds = "{0:02d}".format(ret.seconds % 60)
+    return "{0}:{1}:{2}".format(hours, minutes, seconds)
```

Comment:
I think we can just print `ret` here rather than coming up with our own format.

If printing `timedelta` object, we will have `6:49:59` or `3 days, 6:49:59` if longer than 1 day.
If using the current implementation, we will have `06:49:59` or `78:49:59` if longer than 1 day, seems better IMO. Then you might also need to calculate `month`, `year`, etc.

Reply:
It is looks weird if a build lasts for over days.